setupTests-integration.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable global-require */
 import * as Path from 'path';
```

Comment:
Maybe import only `parse` since it's the only one used.

Reply:
Good point. Done on [e0b7301](https://github.com/HathorNetwork/hathor-wallet-headless/pull/174/commits/e0b7301ba3e5148be371bb786c9c622919049655)

Reply:
I guess the idea with this import change is that you would be able to remove the comment above that disables the lint check. `/* eslint-disable global-require */`

I think you should remove it now that you've changed the import.

Reply:
Actually this `global-require` is there because of lines 22 and 33, and it's still required.

This issue is being discussed on [a thread below](https://github.com/HathorNetwork/hathor-wallet-headless/pull/174#discussion_r845464136) about the other setup file, but the reasons are the same.